fission/src/ui/panels/configuring/ChooseInputSchemePanel.tsx

```diff
@@ -0,0 +1,160 @@
+import Panel, { PanelPropsImpl } from "@/components/Panel"
+import DefaultInputs from "@/systems/input/DefaultInputs"
+import InputSchemeManager, { InputScheme } from "@/systems/input/InputSchemeManager"
+import InputSystem from "@/systems/input/InputSystem"
+import PreferencesSystem from "@/systems/preferences/PreferencesSystem"
+import SynthesisBrain from "@/systems/simulation/synthesis_brain/SynthesisBrain"
+import Button from "@/ui/components/Button"
+import Label, { LabelSize } from "@/ui/components/Label"
+import { useModalControlContext } from "@/ui/ModalContext"
+import { usePanelControlContext } from "@/ui/PanelContext"
+import { Box, Divider, styled } from "@mui/material"
+import { useEffect, useReducer } from "react"
+import { AiOutlinePlus } from "react-icons/ai"
+import { IoCheckmark, IoTrashBin } from "react-icons/io5"
+
+const ChooseInputSchemePanel: React.FC<PanelPropsImpl> = ({ panelId }) => {
+    const { closePanel } = usePanelControlContext()
+    const { openModal } = useModalControlContext()
+
+    const [_, update] = useReducer(x => !x, false)
+
+    const AddIcon = <AiOutlinePlus size={"1.25rem"} />
+    const DeleteIcon = <IoTrashBin size={"1.25rem"} />
+    const SelectIcon = <IoCheckmark size={"1.25rem"} />
```

Comment:
could you add an edit button as well (or instead of the delete button) for the custom control schemes?